pom.xml

```diff
@@ -21,6 +21,13 @@
     <tag>HEAD</tag>
   </scm>
 
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
```

Comment:
Yeah, Netbeans is very useful for common Jenkins development flows (special Kudos to Jesse for Stapler plugin & Co)